coordinator/tests/model/hl7-v3-datatypes-codes.spec.ts

```diff
@@ -0,0 +1,18 @@
+import * as codes from "../../src/model/hl7-v3-datatypes-codes"
+
+describe("GlobalIdentifier", () => {
+  test("root not converted to uppercase when not passed a UUID", () => {
+    const identifier = new codes.GlobalIdentifier("exampleUUID")
+    expect(identifier._attributes.root).toBe("exampleUUID")
+  })
+
+  test("root converted to uppercase when passed a lowercase UUID", () => {
+    const identifier = new codes.GlobalIdentifier("859f8240-d02f-4ea3-8f7f-0aed7b4d7d4e")
+    expect(identifier._attributes.root).toBe("859F8240-D02F-4EA3-8F7F-0AED7B4D7D4E")
```

Comment:
might be cooler to separate the Id string out into a variable and run `var.toUppercase()` on it rather than writing it in uppercase both times